modules/effects/spec/types/effect_creator.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('createEffect()', () => {
       expectSnippet(`
         const effect = createEffect(() => ({ foo: 'a' }), { dispatch: false });
       `).toFail(
-        /Type '{ foo: string; }' is not assignable to type 'Observable<{}> | ((...args: any[]) => Observable<{}>)'./
+        /Type '{ foo: string; }' is not assignable to type 'Observable<unknown> | ((...args: any[]) => Observable<unknown>)'./
```

Comment:
I have to take a look why this happens

Reply:
Did you sync? :) 
https://github.com/ngrx/platform/commit/f70600f5caa12694f48b519c051c449fe57bc556

Reply:
This seems to be because TS is upgraded.
It only occurs when a non-observable is returned.

For a non-dispatching effect that's returning an observable, the effect has the correct returntype:

```
loginSuccess$ = createEffect(
    () =>
      this.actions$.pipe(
        ofType(AuthApiActions.loginSuccess),
        tap(() => this.router.navigate(['/']))
      ),
    { dispatch: false }
  );
```

Returns
```
AuthEffects.loginSuccess$: Observable<{
    user: User;
} & TypedAction<"[Auth/API] Login Success">> & CreateEffectMetadata
```